test/built-ins/Temporal/PlainDateTime/prototype/subtract/casts-argument.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
Same comments for this file as for the one under `add/`.

Reply:
done